app/Http/Controllers/PageController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Page;
+use Illuminate\View\View;
+
+class PageController extends Controller
+{
+    public function index(): View
+    {
+        $pages = Page::where('is_published', true)
+            ->orderBy('updated_at', 'desc')
+            ->get();
+
+        return view('pages.index', [
+            'pages' => $pages,
+        ]);
+    }
+
+    public function show(Page $page): View
+    {
+        $this->authorize('view', $page);
```

Comment:
Is this the common way to authorize in a controller?

Would it be better to reference the gate / policy in the web routes?